.github/CODEOWNERS

```
@@ -0,0 +1 @@
+docs/style-guide.rst @fulldecent
```

This apparently follows `.gitignore` rules:

> A CODEOWNERS file uses a pattern that follows most of the same rules used in [gitignore](https://git-scm.com/docs/gitignore#_pattern_format) files

So I think you need an initial slash unless you want to match any `docs/style-guide.rst` at any nesting level in the repo.
```suggestion
/docs/style-guide.rst @fulldecent
```